### ISSUE TITLE
PNG RGB8 -> BMP RGB8 saving is not working

### ISSUE THREAD

**Issue by rbt2008** [GPUOpen-Tools/Compressonator#14]
I'm using Compressonator CLI.

Following command:

```
./CompressonatorCLI.exe -fs RGB_888 -fd RGB_888 testpat_y_optimized.png qwe.bmp
```

on this texture: https://sites.google.com/site/richgel99/_/rsrc/1468890589308/luma_chroma_texture_compression/testpat.1k.png_y_optimized.png

will result in uncomplete image saving:
http://i.imgur.com/XQTFuFZ.png

**Comment by NPCompress** [comment]
We are investigating the issue and will report back ASAP on the issue.

**Comment by xooiamd** [comment]
please try with CompressonatorCLI.exe -fs ARGB_8888 -fd ARGB_8888 testpat_y_optimized.png qwe.bmp This should give you the complete image saving.

**Comment by NPCompress** [comment]
Hi  rbt2008,

Did the suggestion from xooiamd help with your issue?

**Comment by rbt2008** [comment]
@NPCompress 

Setting input and output formats to **ARGB_8888** to do the trick, thanks!. But I can't understand - there is no alpha channel in png and bmp files. Will this be fixed in the upcoming releases?

Thanks!

**Comment by NPCompress** [comment]
We will take a look at the data of the sample file and verify alpha is preserved during the transfer from png to bmp.

**Comment by quiret** [comment]
BMP with alpha channel seems pretty non-standard.

**Comment by NPCompress** [comment]
Full channel mapping will be available in V2.5 release.
